modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/viewmodel/OnboardingUpgradeFeaturesViewModel.kt

```diff
@@ -102,7 +102,7 @@ class OnboardingUpgradeFeaturesViewModel @Inject constructor(
         val showNotNow = source == OnboardingUpgradeSource.RECOMMENDATIONS
 
         val upgradeLayout = when {
-            FeatureFlag.isEnabled(Feature.PAYWALL_AB_EXPERIMENT) -> UpgradeLayout.Features
+            FeatureFlag.isEnabled(Feature.PAYWALL_AB_EXPERIMENT) && !showPatronOnly -> UpgradeLayout.Features
```

Comment:
Previously the code check `FeatureFlag.isEnabled(Feature.PAYWALL_AA_EXPERIMENT)` only if `Feature.PAYWALL_AB_EXPERIMENT` is disabled. Now it will check that condition also if `showPatronOnly` is true. Is this correct?

Reply:
I reverted this change to address in another PR. The issue is already created here: https://github.com/Automattic/pocket-casts-android/issues/2719